src/Data/Quaternion/Rotation.purs

```diff
@@ -107,3 +112,24 @@ act (Rotation p) v =
   unsafePartial $ case Vec.toArray v of
     [x, y, z] ->
       vectorPart (Quaternion zero x y z `conjugateBy` p)
+
+normalize :: Rotation Number -> Rotation Number
```

Comment:
Right, makes sense, thanks. In that case could you please add a doc-comment? If we said something like "even though functions like `fromQuaternion` already ensure that the quaternion inside a `Rotation` is normalised, `Rotation` values can drift after a sufficiently large number of arithmetic operations; in this case, the `normalize` function can be used to rescale the quaternion back to one with norm 1" do you think that would work?